scipy/stats/stats.py

```diff
@@ -7289,17 +7289,21 @@ def _square_of_sums(a, axis=0):
         return float(s) * s
 
 
-def rankdata(a, method='average'):
+def rankdata(a, method='average', axis=None):
```

Comment:
I think we should add `, *, ` before the new keyword to make it keyword only:
```
def rankdata(a, method='average', *, axis=None):
```

Reply:
OK. Just to add extra protection for the user against themselves?
Interesting to learn about that use of `*` though. Haven't noticed it before.